notebooks/02_model/lightfm_deep_dive.ipynb

```diff
@@ -0,0 +1,1505 @@
+{
```

Comment:
<p>in other notebooks what we have is a step where we perform the predicion and then another step/cell where we compute the metrics (precision_at_k, recall, etc.) </p><p> </p><p>Then it would be great that on a different cell we add the comparison between fit and fit_partial </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Thanks, now it contains sections on how to get the data into the right format for the repo's metrics.